plugins/repository-hdfs/hadoop-client-api/build.gradle

```diff
@@ -8,3 +8,9 @@ dependencies {
 tasks.named('shadowJar').configure {
   exclude 'org/apache/hadoop/util/ShutdownHookManager$*.class'
 }
+
+['jarHell', 'thirdPartyAudit', 'forbiddenApisMain', 'splitPackagesAudit'].each {
```

Comment:
All kinds of problems here that have been not discovered before. 
TODO: raise ticket